sql/updates/world/spell_script_name_heroic_leap.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM `spell_script_names` WHERE `spell_id` IN ('6544', '178368');
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+('6544','spell_warr_heroic_leap'),
+('178368','spell_warr_heroic_leap_jump');
```

Comment:
Add a newline/linefeed here. While you are on it, rename the file to something like 2015_XX_XX_NN_world.sql